app/models/filters/filter_by_repeatedness.rb

```diff
@@ -6,18 +6,18 @@ class FilterByRepeatedness < Filter
     validates :repeated_classifications, inclusion: {in: REPEATED_CLASSIFICATIONS}
 
     def apply(extract_groups)
+      ordered_extract_groups = extract_groups.sort_by(&:classification_at)
       case repeated_classifications
       when "keep_all"
         extract_groups
```

Comment:
do you want to ensure these are returned in the new order or it doesn't matter so you can still use `extract_groups`?